website/content/en/preview/getting-started/migrating-from-cas/scripts/step08-generate-chart.sh

```diff
@@ -0,0 +1,7 @@
+helm template --namespace karpenter \
+    karpenter karpenter/karpenter \
+    --set aws.defaultInstanceProfile=KarpenterInstanceNodeRole \
+    --set clusterEndpoint="${OIDC_ENDPOINT}" \
+    --set clusterName=${CLUSTER_NAME} \
+    --set serviceAccount.annotations."eks\.amazonaws\.com/role-arn"="arn:aws:iam::${ACCOUNT_NAME}:role/KarpenterController" \
+    --version {{< param "latest_release_version" >}} > karpenter.yaml
```

Comment:
this won't render correctly now that it's in a script :(  

We worked around this in the eksctl getting started guide by having an:

```
export KARPENTER_VERSION={{< param "latest_release_version" >}}
```

at the top of the Getting Started Guide (not in a script) and using that env var in the scripts to keep the versioned docs working.